tools/ubuntu-setup/pip.sh

```diff
@@ -1,4 +1,3 @@
 sudo apt-get install -y python-pip
 sudo pip install jsonschema
 sudo pip install argcomplete
-sudo pip install docker-py
```

Comment:
That's what I suspected, so why docker-py not installed before ansible? or at least make a comment if it doesn't matter  the order of the installation.